push_manifest.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+IMAGE_NAME=${1:-}
+IMAGE_TAG=${2:-latest}
+DOCKER_REPOSITORY=${3:-cilium}
+DOCKER_REGISTRY=${4:-}
+
+
+IMAGE_ARCH=("amd64" "arm64")
+
+if [ -z "${IMAGE_NAME}" ]
+then
+  echo "Please specify a image name!"
+  echo -e "\nUsage::\n\tpush_manifest.sh IMAGE_NAME [IMAGE_TAG] [DOCKER_REPOSITORY] [DOCKER_REGISTRY]"
+  echo -e "\nExample::\n\tpush_manifest.sh cilium-runtime latest"
+  exit 1
+fi
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
Is there something below that is relying on some functionality in a particular version of docker?

Reply:
As you mentioned, other engines may be used, so a separate tool is used for manifest operations.